arrow-core/src/main/kotlin/arrow/core/extensions/const/divide/ConstDivideInstance.kt

```diff
@@ -30,8 +26,8 @@ import kotlin.jvm.JvmName
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-  "divide(MO, arg0, arg1, arg2)",
-  "arrow.core.Const.divide"
+    "Cons.mapN(MO, arg0, arg1).map(arg2)",
+    "arrow.core.Cons"
```

Comment:
```suggestion
    "Cons.mapN(MO, arg0, arg1) { a, b -> arg2(Tuple2(a, b)) }",
    "arrow.core.Cons", "arrow.core.Tuple2"
```

Same for other `divide` methods.